sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6462 @@
+                    "hideLabel": false,
+                    "tabindex": "",
+                    "disabled": false,
+                    "autofocus": false,
+                    "dbIndex": false,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": false,
+                    "attributes": {},
+                    "validateOn": "change",
+                    "conditional": {
+                        "show": null,
+                        "when": null,
+                        "eq": ""
+                    },
+                    "overlay": {
+                        "style": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "properties": {},
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "idPath": "id",
+                    "clearOnRefresh": false,
+                    "limit": 100,
+                    "dataSrc": "values",
+                    "valueProperty": "",
+                    "lazyLoad": true,
+                    "filter": "",
+                    "searchEnabled": true,
+                    "searchDebounce": 0.3,
+                    "searchField": "",
+                    "minSearch": 0,
+                    "readOnlyValue": false,
+                    "authenticate": false,
+                    "ignoreCache": false,
+                    "template": "<span>{{ item.label }}</span>",
+                    "selectFields": "",
+                    "selectThreshold": 0.3,
+                    "uniqueOptions": false,
+                    "fuseOptions": {
+                        "include": "score",
+                        "threshold": 0.3
+                    },
+                    "indexeddb": {
+                        "filter": {}
+                    },
+                    "customOptions": {},
+                    "useExactSearch": false,
+                    "id": "e3axg4b"
+                },
+                {
+                    "label": "Level of study code",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "customClass": "",
+                    "modalEdit": false,
+                    "defaultValue": null,
+                    "persistent": true,
+                    "protected": false,
+                    "dbIndex": false,
+                    "encrypted": false,
+                    "redrawOn": "",
+                    "customDefaultValue": "",
+                    "calculateValue": "value = data.levelOfStudyCodes.filter((d) => d.credentialType === data.credentialType)[0][\"levelOfStudyCode\"];\n",
```

Comment:
Do we need this piece of logic here to simplify some other method?

Reply:
I have updated it since we no longer need to filter.

Reply:
My question was more about why we need it at all. Can we remove it?

Reply:
Removed